scripts/apps/search/directives/ItemList.js

```diff
@@ -1950,6 +1967,10 @@ export function ItemList(
                     }
                 });
 
+                if (listConfig.thinRows) {
+                    scope.$on('item:previewed', listComponent.setNarrowView);
```

Comment:
imo there is no need to put here this `if`, in case this is not configured it shouldn't fire that event, so it seems like duplication